appengine/src/main/java/com/google/googleinterns/gscribe/models/ExamInstance.java

```diff
@@ -118,7 +118,7 @@ public boolean equals(Object o) {
                 getUserID().equals(that.getUserID()) &&
                 Objects.equals(getStartTime(), that.getStartTime()) &&
                 Objects.equals(getEndTime(), that.getEndTime()) &&
-                getAnswers().equals(that.getAnswers());
+                Objects.equals(getAnswers(), that.getAnswers());
```

Comment:
Don't use a getter within the same class.

Reply:
Updated.